.env

```diff
@@ -1,9 +1,6 @@
-###> symfony/framework-bundle ###
 APP_ENV=dev
 APP_SECRET=b2c78bf3e68a80912087a764c89bf7c7
-###< symfony/framework-bundle ###
-
-AKENEO_API_BASE_URI=
+AKENEO_API_BASE_URI=http://httpd:80
```

Comment:
Should we let it as is ?

Reply:
We should do a better README instead, yes.

Reply:
YES

Reply:
YES